### ISSUE TITLE
vscode exit on powershell exit

### ISSUE THREAD

**Issue by nueko** [lukesampson/scoop-extras#434]
run `code /path/to/folder` and exit the powershell will exit the visual studio code.

edit: won't exit if the instance already running.

**Comment by r15ch13** [comment]
Using the `bin\code.cmd` fixes this problem without adding the directory to PATH.

**Comment by nueko** [comment]
Nice catch!

**Comment by inosik** [comment]
> Using the bin\code.cmd fixes this problem without adding the directory to PATH.

This won't work for Cygwin/MinGW users, because Bash doesn't pick up `.cmd` shims. See lukesampson/scoop#1237. We'd have to use `code.cmd /path/to/project`.

BTW, I can't reproduce the original issue with VS Code 1.12.1.

**Comment by r15ch13** [comment]
The original issue only occurs only when using Powershell. I couldn't reproduce it in Cmder (using PowerShell).

**Comment by nueko** [comment]
should we add the directory to PATH?

**Comment by inosik** [comment]
@r15ch13 That makes sense. I was using PowerShell in ConsoleZ.

@nueko If we want this behavior (launch VS Code, don't wait for exit), then yes. But other apps have the same behavior as VS Code had before this change (I tried Notepad2 and Notepad++).

What I always do in Bash to launch VS Code without waiting for it to exit, is this:

```shell
code . > /dev/null &
```

Does PowerShell support something similar to start a background process?

We could also try to find a way to create `.exe` shims for `.cmd` files. This would require to change Scoop.

I also tried to change the manifest to this:

```json
"bin": [
  "code.exe",
  [ "cmd", "vscode.exe", "/c", "bin\\code.cmd" ]
]
```

But it fails with `Can't shim 'cmd': File doesn't exist.`
